modules.d/45plymouth/module-setup.sh

```diff
@@ -12,7 +12,8 @@ pkglib_dir() {
 read_setting_from_config() {
     local setting="$1"
     local config_file="$2"
-    sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
+    [[ -f $config_file ]] \
+        && sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
```

Comment:
For better readability I suggest to use if-then-else:

```suggestion
    if [[ -f $config_file ]]; then
        sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
    fi
```